Fix AffineTransfrom3D Inverse Transform Bug

awalter17 opened the pull request:
Hello!

I noticed that after calling `rotate(..)`, `scale(...)`, or `translate(...)` on an `AffineTransform3D` the corresponding inverse transform matrix was not being updated. These changes add code to update the inverse transform after those method calls, and adds tests to ensure the inverse is updated. 

Please feel free to let me know if there are any issues with these commits.

Thanks!

axtimwalde commented:
Thanks,

how about

https://github.com/imglib/imglib2-realtransform/blob/a1f6b3ad26711e681587354c4adc1f86da0d209d/src/main/java/net/imglib2/realtransform/AffineTransform3D.java#L699-L709

?

awalter17 commented:
Missed that one! Good catch @axtimwalde! I've updated `setTranslation` and added an additional check to the tests. Please let me know if there's any additional issues. :)

axtimwalde commented:
Great!  Thanks!
